commerce_coordinator/apps/orders/tests.py

```diff
@@ -0,0 +1,102 @@
+                                                                              'value': True}],
+                                                        'expires': '2022-11-08T22:54:30.777313Z',
+                                                        'id': 3,
+                                                        'is_available_to_buy': True,
+                                                        'price': '149.00',
+                                                        'product_class': 'Seat',
+                                                        'stockrecords': [{'id': 3,
+                                                                          'partner': 1,
+                                                                          'partner_sku': '8CF08E5',
+                                                                          'price_currency': 'USD',
+                                                                          'price_excl_tax': '149.00',
+                                                                          'product': 3}],
+                                                        'structure': 'child',
+                                                        'title': 'Seat in edX Demonstration '
+                                                        'Course '
+                                                        'with verified certificate (and '
+                                                        'ID '
+                                                        'verification)',
+                                                        'url': 'http://localhost:18130/api/v2/'
+                                                        'products/3/'},
+                                            'quantity': 1,
+                                            'status': 'Complete',
+                                            'title': 'Seat in edX Demonstration             '
+                                            'Course with verified certificate (and ID '
+                                            'verification)',
+                                            'unit_price_excl_tax': '149.00'}],
+                                'number': 'EDX-100004',
+                                'payment_processor': 'cybersource-rest',
+                                'status': 'Complete',
+                                'total_excl_tax': '149.00',
+                                'user': {'email': 'edx@example.com', 'username': 'edx'},
+                                'vouchers': []}]}
+
+
+class OrderRetrievalTests(TestCase):
+    """
+    Verify endpoint availability for order retrieval endpoint(s)
+    """
+    maxDiff = None
+
+    def setUp(self):
+        super().setUp()
+        self.mock_response = sample_response
+
+    @patch('commerce_coordinator.apps.orders.clients.EcommerceApiClient.get_orders')
+    def test_EcommerceApiClient(self, mock_response):
```

Comment:
It can be confusing in terms of consistency throughout the whole project to use both underscore and camel case in a function name?

Reply:
https://peps.python.org/pep-0008/#function-and-variable-names
^ Not sure if it applies to test functions

Reply:
goooooood question..... I'd imagined I should use the name of the thing I'm testing (which has a camel case name), but you're right: I'm looking at examples in ecommerce it looks like we keep camel case for the test class name (if applicable) and understores for test names- good catch! Update coming...